iphone/Classes/TiUITextWidget.m

```diff
@@ -160,6 +160,70 @@ -(void)setAutocorrect_:(id)value
 	[[self textWidgetView] setAutocorrectionType:[TiUtils boolValue:value] ? UITextAutocorrectionTypeYes : UITextAutocorrectionTypeNo];
 }
 
+-(void)setTextContentType_:(id)value
+{
+    ENSURE_TYPE_OR_NIL(value, NSString);
+    if (@available(iOS 10.0, *)) {
```

Comment:
This whole method should be rewritten, since it has way too much manual logic and some missing guard inside it. Let try this something like this:
```objc
- (void)setTextContentType_:(id)value
{
    ENSURE_TYPE_OR_NIL(value, NSString);

    if (![TiUtils isiOS10OrGreater]) {
        NSLog(@"Setting a text-content-type is only available on iOS 10 and later");
        return;
    }

    [[self textWidgetView] setTextContentType:[TiUtils stringValue:value]];    
}
```

This has some advantages:
- Constants are mapped to it's values automatically
- There is no need for the Xcode 9-only `@available` syntax
- There is no need for the (currently missing) `#if IS_XCODE_9` macros
- iOS 11 constants do not need to be checked manually.
- Less error-prone due to no manual validation